index.html

```diff
@@ -51,21 +51,11 @@
       <!-- Menu end -->
 
       <!-- Highlight section  -->
-      <div class="highlight-container">
-        <div class="highlight-big"></div>
-        <div class="highlight-side-container">
-          <p class="highlight-title">game title</p>
-          <div
-            class="highlight-medium sm:flex sm:w-52 sm:flex-col md:flex md:w-full md:flex-row"
-          >
-            <div class="highlight-small sm:h-full"></div>
-            <div class="highlight-small sm:h-full"></div>
-          </div>
-          <div class="highlight-medium md:flex md:flex-row">
-            <div class="highlight-small"></div>
-            <div class="highlight-small"></div>
-          </div>
-        </div>
+      <div class="highlight-display">
+        <div class="highlight-container"></div>
+        <div class="highlight-container hidden sm:block"></div>
```

Comment:
Having the same element multiple times hidden only to show it on certain break points is not ideal
Especially since this is a container